README.md

```diff
@@ -28,3 +30,14 @@ I'm hoping to get the current ddev-webserver-dev test converted to [BATS](github
 | DDEV-LIVE PHP with just php7.3 | 752MB | 594MB |
 | ddev-live-nginx-prod | 136MB  |   181MB    |                                                 |
 | ddev-webserver-prod (new hardened open source image) | 1.01GB | |
```

Comment:
should the columns here be swapped?

`ddev-webserver-prod (new hardened open source image) | N/A | 1.01GB |`